lib/src/teledart/fetch/long_polling.dart

```diff
@@ -108,9 +108,17 @@ class LongPolling extends AbstractUpdateFetcher {
     return Future.value();
   }
 
-  void _onRecursivePollingHttpError(HttpClientException error) {
+  Future<void> _onRecursivePollingHttpError(HttpClientException error) async {
     if (error.isTooManyRequests()) {
       retryDelay = error.parameters?.retryAfter_ ?? retryDelay;
+      _onRecursivePollingError(error);
+    } else if (error.code == 409) {
```

Comment:
[409 error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) is a conflict error, and it does not seem to be a retry-able error.

Reply:
Okay, but like in 429 error, it must be a way to catch it and not let the whole app crash

Reply:
this [PR](https://github.com/DinoLeung/TeleDart/pull/243/files) has export the http client along with the exception class 👍